src/Container.php

```diff
@@ -21,6 +21,11 @@ class Container implements ContainerInterface, ArrayAccess
     protected $items = [];
 
     /**
+     * @var array - \League\Container\ServiceProvider[]
```

Comment:
No but the description here is - I think lol.

What are you trying to do? I think you’re misusing the description and dunno why the - is there

-- 
Phil

From: Phil Bennett <notifications@github.com>
Reply: thephpleague/container <reply@reply.github.com>>
Date: March 2, 2015 at 3:12:35 PM
To: thephpleague/container <container@noreply.github.com>>
Cc: Phil Sturgeon <me@philsturgeon.uk>>
Subject:  Re: [container] Introduce Service Provider Functionality (#18)  

In src/Container.php:

> @@ -21,6 +21,11 @@ class Container implements ContainerInterface, ArrayAccess
>      protected $items = [];
>   
>      /**
> +     * @var array - \League\Container\ServiceProvider[]
Should still just parse the arrayof class name as description shouldn't it? The type just being array. Actually never tested that, does the description HAVE to be above on properties?

—
Reply to this email directly or view it on GitHub.